Makefile

```diff
@@ -27,3 +30,11 @@ clean:
 	$(MDTOOL) build -t:Clean ModernHttpClient.sln
 	rm *.dll
 	rm -rf build
+	
```

Comment:
great helper method, I don't think makefile should have dependancies on things like xmlstarlet

Reply:
I've removed the version-bump target